api/README.md

```diff
@@ -11,14 +11,20 @@ If you use MongoDB Atlas, the set is managed for you.
 ### Local
 
 The simplest way to run a replica set locally is to use the docker-compose file
-in /tools. First disable any running MongoDB instance on your machine, then run
-the docker-compose file.
+in /tools. Once that's running, update the connection string in .env to use port 27018.
 
 ```bash
 cd tools
 docker compose up -d
 ```
 
```

Comment:
````suggestion
in /tools.

```bash
cd tools
docker compose up -d
```

Once that's running, update the connection string in the `.env` file to use port `27018`.

```txt
# Database
MONGOHQ_URL=mongodb://127.0.0.1:27017/freecodecamp?directConnection=true
```

````

Just for clarity?